open-api/immich-openapi-specs.json

```diff
@@ -6221,7 +6221,7 @@
   "info": {
     "title": "Immich",
     "description": "Immich API",
-    "version": "1.93.0",
+    "version": "1.93.1",
```

Comment:
I don't think this PR should update this

Reply:
It got updated automatically with `git checkout main`